lib/http-bridge/test/bench/Main.hs

```diff
@@ -315,9 +316,16 @@ getCurrentSlot net = calcSlot <$> startTime net <*> getPOSIXTime
         idx = floor ((d - (fromIntegral ep) * epochDur) / slotDur)
 
     startTime :: MonadFail m => Text -> m POSIXTime
-    startTime "mainnet" = pure 1506203091
-    startTime "testnet" = pure 1537941600
```

Comment:
This second value (for "testnet") seems to differ from the value defined in `lib/http-bridge/src/Cardano/Wallet/HttpBridge/Compatibility.hs`:
```hs
byronBlockchainParameters 
    :: forall n. KnownNetwork n   
    => BlockchainParameters (HttpBridge n)
byronBlockchainParameters = BlockchainParameters  
    { getGenesisBlock = block0    
    , getGenesisBlockDate = case networkVal @n of 
        Mainnet -> StartTime $ posixSecondsToUTCTime 1506203091   
        Testnet -> StartTime $ posixSecondsToUTCTime 1563999616
```

Reply:
As explained in the PR's comments, that was precisely the reason for the failure of the nightly builds. These times were hard-coded in two places and we only updated the one in `.../Compatibility.hs`. Now, the benchmark code uses also `../Compatibility.hs` which should fix the discrepancy and future similar issues